actors/providers/ldap/pom.xml

```diff
@@ -65,11 +65,13 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <artifactId>orbit-actors-json</artifactId>
             <version>${project.version}</version>
         </dependency>
+
         <dependency>
-            <groupId>org.apache.directory.api</groupId>
-            <artifactId>api-all</artifactId>
-            <version>1.0.0-M30</version>
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-all</artifactId>
```

Comment:
instead of including apacheds-all with the entire ldap module it could be limited to test scope, right? then perhaps api-ldap-client-api for the ldap module itself to keep it lean.